src/types.jl

```diff
@@ -270,9 +192,6 @@ for T in [:Arb, :Acb], A in [:Vector, :Matrix]
     end
 end
 
-const ArbLike = Union{Arb,ArbRef,Ptr{arb_struct},arb_struct}
-const AcbLike = Union{Acb,AcbRef,Ptr{acb_struct},acb_struct}
-
```

Comment:
where did these go?

Reply:
They had duplicate definitions, the are defined again further down in the file. I mentioned this in the commit message but I have realised that those messages mostly get lost because at least I basically only look at the total diff under "Files changed" where they are not shown...